### PR TITLE
feat: backport v1.27 changes to v1.26

### PR DESCRIPTION
**Pull request by alessiodionisi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +59,7 @@ Core modules provide essential functionality to the distribution for production-
 | [Ingress][ingress-module]       | ![Version][ingress-version]    | Fast and reliable Ingress Controller and TLS certificate management                       |
 | [Logging][logging-module]       | ![Version][logging-version]    | A centralized logging solution based on the OpenSearch stack                              |
 | [Monitoring][monitoring-module] | ![Version][monitoring-version] | Monitoring and alerting functionality based on Prometheus, AlertManager and Grafana       |
+| [Monitoring][tracing-module] | ![Version][tracing-version] | Tracing functionality based on Tempo |
```
Comment (Al-Pragliola) [inline README.md:62]: Tracing*

**Review by Al-Pragliola — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (alessiodionisi) [reply README.md:62]: Also on 1.27

Comment (omissis) [reply README.md]: fixed

**Review by Al-Pragliola — APPROVED** [review]
lgtm

**Pull request merged**